internal/nodes/avi_model_nodes.go

```diff
@@ -775,6 +775,11 @@ func (v *AviVsNode) CalculateCheckSum() {
 	if v.EnableRhi != nil {
 		checksum += utils.Hash(utils.Stringify(*v.EnableRhi))
 	}
+	for _, vsvipref := range v.VSVIPRefs {
```

Comment:
Why should we update the VS here ? Won't VSVIP be enough ?

Reply:
not required, removed it